machine-learning/ml/preprocessing/scaling.py

```diff
@@ -13,6 +13,21 @@ def perform_scaling(x):
     scaler = MinMaxScaler()  # Default behavior is to scale to [0,1]
     columns = x.columns
     x = scaler.fit_transform(x)
-    x = pd.DataFrame(x, columns=columns) # keeping the column names
+    x = pd.DataFrame(x, columns=columns)  # keeping the column names
 
     return x, scaler
+
+
+def perform_scaling(x, scaler):
```

Comment:
maybe this method should not return the scaler, as it uses it but does not produce it.

Reply:
Changed it